test/functional/cmdLineTests/callsitedbgddrext/playlist.xml

```diff
@@ -30,7 +30,7 @@
 		</variations>
 		<command>$(MKTREE) $(REPORTDIR); \
 	cd $(REPORTDIR); \
-	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl zos ; \
+	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl all zos ; \
```

Comment:
"all zos" is inconsistent with the later calls which specify "all"

Reply:
`all` is passed in to the `sysvcleanup.pl` script to ensure data from all users is cleaned up. AIX and ZOS have different user bits that are set by `sysvcleanup.pl` so the second option `zos` is passed when the test target is AIX/ZOS. `sysvcleanup.pl` accepts both `aix` and `zos` as a second option but they are identical.